docs/website/blog/2025-06-XX-client-cli-utxo-hd-ledger-state-snapshot-conversion

```diff
@@ -0,0 +1,28 @@
+---
+title: Mithril client CLI new 'tools utxo-hd snapshot-converter' command
+authors:
+  - name: Mithril Team
+tags: [mithril client, cli, command, utxo-hd, conversion]
+---
+
+### Introducing 'tools utxo-hd snapshot-converter' command
+
+With the release of the new distribution [2524](https://github.com/input-output-hk/mithril/releases/tag/2524.0), we have added the **tools** command in the **client CLI**.
+
+Since the release of the Cardano node [10.1.4](https://github.com/IntersectMBO/cardano-node/releases/tag/10.4.1), the Cardano database restoration downloads an `InMemory` UTxO-HD ledger state snapshot. However, if your Cardano node is configured for an on-disk format (`LMDB`) or the `legacy` UTxO-HD format, you need to convert that snapshot before running it.
+
+The new `utxo-hd snapshot-converter` subcommand streamlines the Cardano node bootstrapping process by converting the restored ledger state snapshot into the expected flavor (on-disk `LMDB` or `Legacy`). It uses the `snapshot-converter` binary under the hood, bundled with the Cardano node distribution.
+
+This conversion tool is `unstable` and is available for Linux, macOS and Windows.
```

Comment:
This is the command which is unstable not the `snapshot-converter` binary. Can you mke that a bit more clear here?